packages/core/src/integrations/inboundfilters.ts

```diff
@@ -131,7 +131,6 @@ function _isIgnoredTransaction(event: Event, ignoreTransactions?: Array<string |
 }
 
 function _isDeniedUrl(event: Event, denyUrls?: Array<string | RegExp>): boolean {
-  // TODO: Use Glob instead?
```

Comment:
@AbhiPrasad I think this is not really relevant anymore, is it? 🤔

Reply:
Yeah we can remove!